tests/locator_test.go

```diff
@@ -398,7 +398,11 @@ func TestLocatorElementState(t *testing.T) {
 			"IsDisabled", func(l *common.Locator, tb *testBrowser) { l.IsDisabled(timeout(tb)) },
 		},
 		{
-			"IsHidden", func(l *common.Locator, tb *testBrowser) { l.IsHidden(timeout(tb)) },
+			"IsHidden", func(l *common.Locator, tb *testBrowser) {
+				if _, err := l.IsHidden(timeout(tb)); err != nil {
+					panic(err)
```

Comment:
Can we propagate this with an error instead of a panic?

Reply:
This is a temporary change. This sub test is removed in https://github.com/grafana/xk6-browser/pull/1110.